tests/src/Kernel/Paragraphs/MediaParagraphsTest.php

```diff
@@ -922,4 +927,152 @@ public function testIframe(): void {
     $this->assertStringContainsString('http://example.com/iframe_bg', $iframe->attr('src'));
   }
 
+  /**
+   * Test Carousel paragraph rendering.
+   */
+  public function testCarousel(): void {
+    // Set image media translatable.
+    $this->container->get('content_translation.manager')
+      ->setEnabled('media', 'image', TRUE);
+    // Make the image field translatable.
+    $field_config = $this->container->get('entity_type.manager')
+      ->getStorage('field_config')
+      ->load('media.image.oe_media_image');
+    $field_config->set('translatable', TRUE)->save();
+    $this->container->get('router.builder')->rebuild();
+
+    // Create English files.
+    $en_file_1 = file_save_data(file_get_contents(\Drupal::service('extension.list.theme')->getPath('oe_theme') . '/tests/fixtures/example_1.jpeg'), 'public://example_1_en.jpeg');
```

Comment:
file_save_data() already deprecated function. Should be used Use \Drupal\file\FileRepositoryInterface::writeData(). True for other places.

Reply:
It doesn't work on lowest versions: https://drone.fpfis.eu/openeuropa/oe_theme/7246/33